fedn/cli/project_cmd.py

```diff
@@ -68,33 +68,32 @@ def create_project(ctx, name: str = None, description: str = None, protocol: str
 
     if _token:
         headers["Authorization"] = _token
-    if not no_interactive:
-        if name is None:
-            name = input("Please enter a project name: ")
-        if description is None:
-            description = input("Please enter a project description (optional): ")
-    else:
-        if name is None:
+    if name is None:
+        if no_interactive:
             click.secho("Project name is required.", fg="red")
             return
-        if description is None:
-            description = ""
-    if len(name) > 46 or len(description) >= 255:
+        name = input("Please enter a project name: ")
+    if len(name) > 46:
         click.secho("Project name or description too long.", fg="red")
     else:
         # Call the authentication API
         try:
-            requests.post(url, data={"name": name, "description": description}, headers=headers)
+            response = requests.post(url, data={"name": name}, headers=headers)
+            response_project = response.json().get("project")
+            if not no_header:
```

Comment:
What I mean by --no-hearer is that the header of the table i.e "Name | ID | Status| etc" is not printed but the data is. In other words the rows with actual data